### PR TITLE
Fix #36245 adultmult.xyz

### LINKED ISSUE #36245: adultmult.xyz
### Issue URL (Ads)
[http://adultmult.xyz/other/final_space_2_season.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fadultmult.xyz%2Fother%2Ffinal_space_2_season.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Mac
AdGuard version: | 2.0.6.559 release
Browser: | Safari
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard German

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36245

### CODE REVIEW

File: RussianFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1,6 +1,9 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/36245
+adultmult.xyz##.title_h2:has( > a[href^="https://bp2.infiplay.com/"])
+adultmult.xyz##div:has( > a[href^="https://www.vsemayki.ru/"])
```
Comment (BlazDT) [inline RussianFilter/sections/css_extended.txt]: Hiding the link is enough, it does not save any space when I remove the div too.

For the other two ext-CSS rules, please use #?#

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/css_extended.txt (hunk 1 above)
Comment (slavaleleka) [reply RussianFilter/sections/css_extended.txt]: I suppose that `##a[target="_blank"][rel="nofollow"] > img` can be even better in this case, isn't it?

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -18,6 +18,8 @@ Ly9hbi55YW5kZXgucnUv
 ! adbetnet domains
 ||cdn.*/libs/b.js$third-party,domain=nnm-club.lib|nnm-club.me|nnm-club.tv|nnmclub.to|seasonvar.ru|mintmanga.com|realmadrid.ru
 !
+adultmult.xyz##a[href^="https://bp2.infiplay.com/"]
+adultmult.xyz##a[href^="https://www.vsemayki.ru/"]
```
Comment (BlazDT) [inline RussianFilter/sections/specific.txt]: Last comment was only for rule with ".vsemayk"

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 2 above)
Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: Check if existing adultmult.net rules are works here

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**